src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -0,0 +1,204 @@
+package com.googleinterns.zoomtube.servlets;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.User;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.googleinterns.zoomtube.data.Comment;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.lang.reflect.Type;
+import java.time.Clock;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+public class DiscussionServletTest {
+  private static final int LECTURE_ID = 1;
+  private static final String LECTURE_ID_STR = "1";
+  private static final String TEST_COMMENT_CONTENT = "Test content";
+  private static final String DEFAULT_EMAIL = "test@example.com";
+  private static final String AUTH_DOMAIN = "example.com";
+
+  @Rule public final MockitoRule mockito = MockitoJUnit.rule();
```

Comment:
```suggestion
  @Rule private final MockitoRule mockito = MockitoJUnit.rule();
```

Reply:
`org.junit.internal.runners.rules.ValidationError: The @Rule 'mockito' must be public.`

Reply:
No idea why, or if this can be fixed, but I can't run with it private